python/objects/playerHand.py

```diff
@@ -1,20 +1,129 @@
 import random
+
+from python.logic.straight import Straight
 from python.objects.card import Card
 from python.objects.hand import Hand
 
+
 class PlayerHand(Hand):
-    'A set of two cards'
+    'A representation of the player\'s hand'
 
     def __init__(self):
         self.hand = []
+        self.sortedHand = []
+        self.hole = []
+        self.community = []
+        self.individualScore = 0
+        self.handStrength = 0
+        self.handName = ''
+        self.clubs = 0
+        self.diamonds = 0
+        self.hearts = 0
+        self.spades = 0
+        self.hasFlush = False
+        self.hasStraight = False
+        self.hasStraightFlush = False
         pass
 
     def deal(self, card):
         self.hand.append(card)
+        self.hole.append(card)
+        self.individualScore += card.score
```

Comment:
The individual score only needs to be calculated when we're calculating hand strength.